webpack/.eslintrc

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Better to use the core configurations that use pf-react-eslint

Reply:
using `patternfly-react-eslint` now